server/src/test/java/org/apache/iotdb/db/tools/MLogParserTest.java

```diff
@@ -114,8 +118,10 @@ public void testParseMLog() throws IOException {
     try (BufferedReader reader = new BufferedReader(new FileReader("target" + File.separator
       + "tmp"  + File.separator + "text.mlog"))) {
       int lineNum = 0;
-      while (reader.readLine() != null) {
+      String line;
+      while ((line = reader.readLine()) != null) {
         lineNum++;
+        logger.info(line);
```

Comment:
Do not print log like this... There will be more than 100 lines of log..
(A solution is: do not print the log unless there is an exception thrown.)

Reply:
ok